apps/src/codebridge/InfoPanel/ValidatedInstructions.tsx

```diff
@@ -238,58 +238,47 @@ const ValidatedInstructions: React.FunctionComponent<InstructionsProps> = ({
     if (!hasConditions) {
       return null;
     }
-    return (
-      <div className={moduleStyles['bubble-' + theme]}>
-        {isValidating ? (
-          <Button
-            text={codebridgeI18n.stopValidation()}
-            onClick={handleStop}
-            color={'destructive'}
-            iconLeft={{iconStyle: 'solid', iconName: 'square'}}
-            className={moduleStyles.buttonInstruction}
-            size={'s'}
-          />
-        ) : (
-          <Button
-            text={codebridgeI18n.validate()}
-            onClick={() => handleValidate()}
-            type={'secondary'}
-            disabled={shouldValidateBeDisabled}
-            iconLeft={{iconStyle: 'solid', iconName: 'clipboard-check'}}
-            className={classNames(
-              darkModeStyles.secondaryButton,
-              moduleStyles.buttonInstruction
-            )}
-            color={'white'}
-            size={'s'}
-          />
+    return isValidating ? (
```

Comment:
I'd recommend viewing with whitespace hidden because of the indentation changes in this file.